x-pack/plugins/maps/server/fonts/open_sans/10240-10495.pbf

```diff
@@ -0,0 +1,3 @@
+
+ 
+Open Sans Regular10240-10495
```

Comment:
@nreese what is the role for all these `XXXX-YYYY.pbf` files?

Reply:
Those are the fonts used by mapbox to render text in web gl. @nickpeihl can provide more details if needed.